Classes/ios/Scanners/MTBBarcodeScanner.m

```diff
@@ -716,17 +722,18 @@ - (BOOL)isCapturingStillImage {
 
 #pragma mark - Setters
 
-- (void)setCamera:(MTBCamera)camera {
```

Comment:
This is the setter for the public `camera` property, which lets the user run `scanner.camera = MTBCameraFront` to change the camera. Because switching cameras can return an error, I added `setCamera:error:` to provide more information to the users, but kept this property in for backwards compatibility (and for people who don't care about the error).

I'm OK moving away from the setter method, as people _should_ care about the potential error, but as part of this change we should also remove the public property if that can no longer be used to switch cameras.